### PR TITLE
[zh-cn]: Update translation of Function.name

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

It is the same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15403

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/function/name/index.md [hunk 1]
```diff
@@ -28,196 +34,265 @@ console.log(someFunction.name); // someFunction
...
-### Symbol 作为函数名称
+```js
+class Foo {
+  static name = 123;
+}
+console.log(new Foo().constructor.name); // 123
+```
 
-如果{{jsxref("Symbol")}} 被用于函数名称，并且这个 symbol 具有相应的描述符，那么方法的名字就是方括号中的描述符。
+如果类有一个静态属性名为 `name` ,它也将变为*可写的*。在没有自定义静态定义的情况下,内置定义是*只读的*:
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/name/index.md:257]:
```suggestion
如果类有一个静态属性名为 `name` ,它也将变为*可写的*。在没有自定义静态定义的情况下，内置定义是*只读的*:
```

File: files/zh-cn/web/javascript/reference/global_objects/function/name/index.md [hunk 2]
```diff
@@ -24,200 +30,269 @@ someFunction.name = "otherFunction";
...
+
+### 获取对象的构造函数名称
+
+你可以使用 `obj.constructor.name` 来检查对象的“类”名。
 
 ```js
-function Foo() {}
-Object.defineProperty(Foo, "name", { writable: true });
-Foo.name = function () {};
+function Foo() {} // Or: class Foo {}
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/name/index.md:226]:
```suggestion
function Foo() {} // 或：class Foo {}
```

**Review by yin1999 — COMMENTED** [review]
A great work, just some small nits/suggestions

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**